apps/coscrad-frontend/src/components/resources/media-items/media-item-detail.full-view.presenter.tsx

```diff
@@ -21,7 +21,10 @@ export const MediaItemDetailFullViewPresenter = ({
         >
             <Card className="detail-card">
                 <div id="detail-term" className="detail-meta">
-                    {formatBilingualText(title, titleEnglish)}
+                    <BilingualTextPresenter
```

Comment:
Do media items still have the separate properties? Or were they split above in this file?

Reply:
Just wanting to leave any major changes or re-alignments to a PR dealing with the media items.